docker-compose/database/db-changes/studies/0020_set_default_studycardpolicy.sql

```diff
@@ -0,0 +1 @@
+UPDATE study SET study_card_policy=0;
```

Comment:
I think you have to add the column too as it does not exists on qualif/prod yet.
( ALTER TABLE study ADD COLUMN study_card_policy [...])